src/smart-components/user/user-table-helpers.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import { Label } from '@patternfly/react-core';
+import { CheckIcon, CloseIcon } from '@patternfly/react-icons';
+import messages from '../../Messages';
+import { IntlShape } from 'react-intl';
+
+interface UserProps {
+  email: string;
+  first_name: string;
+  is_active: boolean;
+  is_org_admin: boolean;
+  last_name: string;
+  username: string;
+  uuid: string;
+}
+
+export type CellObject = { title: string | React.RefAttributes<HTMLAnchorElement>; props?: { 'data-is-active': boolean } };
+
+export interface RowProps {
```

Comment:
I see you use this interface to define `TableProps`. We should come up with a more generic solution that would match different data passed to the table.

Reply:
I understand, changing Table prop into Array<unknown> for now